addons/account/models/account_payment_term.py

```diff
@@ -258,7 +258,16 @@ class AccountPaymentTermLine(models.Model):
             ('days_after', 'Days after invoice date'),
             ('days_after_end_of_month', 'Days after end of month'),
             ('days_after_end_of_next_month', 'Days after end of next month'),
+            ('days_end_of_month_on_the', 'Days end of month on the')
```

Comment:
```suggestion
            ('days_end_of_month_on_the', 'Days end of month on the'),
```
To prevent future diff